de.dlr.sc.virsat.model.extension.fdir.ui/src/de/dlr/sc/virsat/model/extension/fdir/ui/diagram/ft/FaultTreeFeatureProvider.java

```diff
@@ -117,6 +118,15 @@ public FaultTreeFeatureProvider(IDiagramTypeProvider dtp) {
 		}
 		
 		createFeatures.add(new BasicEventsCreateFeature(this));
+		
+		//Order create features alphabetically
+		createFeatures.sort(new Comparator<ICreateFeature>() {

```

Comment:
Cool trick with lambdas:

`(feature1, feature2) -> featurre1.getName().compareTo(feature2.getName())`

should do the same. I think the following would be even better and _should_ work

`Comparator.comparing(ICreateFeature::getName())`